modules/tinymce/src/core/test/ts/atomic/html/FilterNodeTest.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import { AstNode } from 'tinymce/core/api/PublicApi';
+import { traverse } from 'tinymce/core/html/FilterNode';
+
+describe('browser.tinymce.core.html.FilterNode', () => {
+  it('TINY-6945: new traverse should work like old recursive traverse', () => {
+    const results: string[] = [];
+
+    const body = new AstNode('body', 11);
+
+    const Child1 = new AstNode('Child1', 1);
```

Comment:
These don't follow our camel case naming (they are PascalCase), please use appropriate names.

- [ ] Fix variable names so they are camel case

Reply:
fixe, I removed the first part and forgot to put the first letter in lowercase sorry :face_exhaling: